scheduler/src/helper/generator.helper.ts

```diff
@@ -0,0 +1,423 @@
+import { config as dotenvConfig } from "dotenv";
+import * as fs from 'fs';
+import { join, resolve } from 'path';
+import { CgiSftpService } from "src/modules/cgi-sftp/cgi-sftp.service";
+import { Transaction } from "src/modules/transactions/transaction.entity";
+import { Readable } from "typeorm/platform/PlatformTools";
+import { CgiConstants } from "src/common/constants/cgi.constant";
+
+const envFilePath = resolve(__dirname, "../../../.env");
```

Comment:
@cberg-aot This is the `generator.helper` file that was failing the other day in @praju-aot's PR build. It was deploying this image where the .env path has changed. This is a variation of the stale image.